wagtail_draftail_anchors/static_src/wagtaildraftailanchors/js/wagtail_draftail_anchor.js

```diff
@@ -107,6 +107,31 @@ window.draftail.registerPlugin({
   decorator: AnchorIdentifier,
 });
 
+const CopyAnchorButton = ({identifier}) => {
+  const [didCopy, setDidCopy] = React.useState(false);
+
+  const copyText = (event) => {
+    // Prevent the button click event from submitting the page form
+    event.preventDefault();
+    navigator.clipboard.writeText(identifier);
+    setDidCopy(true);
```

Comment:
Does `setDidCopy` get reset when you close and reopen the tooltip? I suspect it would, which would be desired since you're likely to stick something else in the clipboard once navigated away, but since I don't have any manual testing time til tomorrow I thought I'd double check with you.

Reply:
Yep, this is the behaviour.